ci_scripts/vm_tools.rb

```diff
@@ -26,15 +26,16 @@ def bring_up_vm_and_wait_for_boot
   booted = false
   until booted
     begin
-      run_command_vm('echo hello world')  
+      run_command_vm('echo hello world')
       booted = true
-    rescue => exception
+    rescue StandardError => e
+      puts e
       puts 'Not online yet, retrying...'
       sleep(10)
     end
   end
   # Copy files to VM
-  run_command_vm("mkdir -p /tmp/build/stripe-ios")
+  run_command_vm('mkdir -p /tmp/build/stripe-ios')
```

Comment:
If there's any chance that old files could be here, would be better to use `mktemp -d`

Reply:
We destroy the VM beforehand, so there shouldn't be